### PR TITLE
Add basic dnsmasq monitoring template and cron configuration

### PR DESCRIPTION
**Pull request by rharrison10**

### CODE REVIEW

**Comment by rharrison10** [comment]
ready to test

**Comment by rharrison10** [comment]
ready to test

**Comment by rharrison10** [comment]
ready to test

**Comment by rharrison10** [comment]
ready to test

**Comment by rharrison10** [comment]
Ready to test

**Comment by rharrison10** [comment]
ok to test

File: ansible/roles/oso_host_monitoring/README.md [hunk 1]
```diff
@@ -14,9 +14,11 @@ Role Variables
 osohm_zagg_web_url: where to contact monitoring service
 osohm_host_monitoring: name of host monitoring container
 osohm_zagg_client: name of container with zabbix client
+osohm_monitor_dnsmasq: Should dnsmasq be monitored on the nodes. `True` or `False`
 osohm_docker_registry_url: docker repository containing above containers
```
Comment (kwoodson) [inline ansible/roles/oso_host_monitoring/README.md:18]: formatting and it looks like there are 2 of these.  Is this right?

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 2]
```diff
@@ -120,6 +123,18 @@ host_monitoring_cron:
...
 
+{% if osohm_host_type == 'node' and osohm_monitor_dnsmasq|bool %}
+{# dnsmasq checks #}
+- name: send dnsmasq process count
+  minute: "*/2"
+  job: ops-runner -f -s 15 -n cspc.openshift.dnsmasq.process.count /usr/bin/cron-send-process-count '^dnsmasq' openshift.dnsmasq.process.count
+
+- name: send openshift-node dnsmasq status every 4 minutes
+  minute: "*/4"
+  job: ops-runner -f -s 50 -n csosms.openshift.dnsmasq /usr/bin/cron-send-os-dnsmasq-checks
```
Comment (kwoodson) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:134]: optional: can make verbose by default.  I think this is the new standard

File: scripts/openshift-tools-scripts.spec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Summary:       OpenShift Tools Scripts
 Name:          openshift-tools-scripts
-Version:       0.0.114
+Version:       0.0.115
```
Comment (kwoodson) [inline scripts/openshift-tools-scripts.spec:3]: I don't think this is needed as tito does version management (I believe).

File: scripts/openshift-tools-scripts.spec [hunk 4]
```diff
@@ -277,6 +279,9 @@ OpenShift Tools AWS specific scripts
 
 
 %changelog
+* Wed Jul 06 2016 Russell Harrison <rharriso@redhat.com> 0.0.115-1
```
Comment (kwoodson) [inline scripts/openshift-tools-scripts.spec]: I think this is handled by tito

**Comment by rharrison10** [comment]
ok to test

**Comment by kwoodson** [comment]
ok to test

**Comment by kwoodson** [comment]
:+1:

**Pull request merged**